CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+### Changed
+* `createpplication` management command enhanced to display an auto-generated secret before it gets hashed.
```

Comment:
typo: `createpplication`

Reply:
oops.

That and other problems with older versions of django....